Code/GraphMol/FileParsers/file_parsers_catch.cpp

```diff
@@ -431,6 +432,620 @@ M  END)CTAB";
   }
 }
 
+TEST_CASE("workaround for broken MJ2009-MJ2011 molblocks",
+          "[feature][sgroups]") {
+  std::string molblock1 = R"CTAB(
```

Comment:
Please put the molblock definition immediately adjacent to the code which tests it (in this case in the `SECTION`s)
Otherwise the reader has to constantly scroll back and forth to figure out what's going on.

Reply:
Will do. The reason why I put them outside sections was to avoid repetitions as the same molblocks are used in different `SECTION`s, but it's not a major problem.

Reply:
The way I solve that is by having multiple tests in one SECTION.
you can create local scopes by just putting the bits that change within curly brackets.

Reply:
Ah, that makes sense. Believe it or not, this is the first `catch` test I write for RDKit!

Reply:
hopefully you find them as delightful to write as I do. :-)
sooooo much better than the old RDKit way of doing things!